rllib/utils/torch_ops.py

```diff
@@ -1,9 +1,14 @@
-import tree
-
 from ray.rllib.utils.framework import try_import_torch
 
 torch, _ = try_import_torch()
 
+try:
+    import tree
+except (ImportError, ModuleNotFoundError):
+    raise ModuleNotFoundError(
```

Comment:
can you instead just `logger.error("...")` and `raise`? Otherwise, this can result in obfuscated error messages.
```
try:
    import tree
except (ImportError, ModuleNotFoundError):
    logger.error('your message')
    raise  #auto re-raise old exception here, don't create a new one
```